.github/settings.yml

```
@@ -43,11 +43,11 @@ repository:
   allow_rebase_merge: false
   allow_squash_merge: false
   default_branch: master
-  description: "GitHub repository template for a PHP package"
+  description: "PHP package to easily convert and work with microplate data"
```

Does not read that smoothly, assuming this is the intent?

"PHP package to easily (convert) and (work with) microplate data"

How about we simplify too:

```suggestion
  description: "PHP package to easily work with microplate data"
```